.gitignore

```diff
@@ -28,6 +28,7 @@ pip-cache
 
 # asv environments, results
 .asv
+.data
```

Comment:
It's misleading to state that _any_ directory called `.data` relates to "`asv environments, results`". We should either:

* Stick this line under a different comment heading.
* Use the specific path (so that would be `benchmarks/.data` I think)

Reply:
[579358c](https://github.com/SciTools/iris/pull/4699/commits/579358c8106b3f5e5b8b947486f835d5af91a7ef)